server/neptune/workflows/internal/activities/github.go

```diff
@@ -121,22 +128,44 @@ func (a *githubActivities) CreateCheckRun(ctx context.Context, request CreateChe
 	}, nil
 }
 
-type GithubRepoCloneRequest struct {
-	Repo           internal.Repo
-	Revision       string
-	DestinationDir string
-	Root           root.Root
+type DownloadRootRequest struct {
+	Repo         internal.Repo
+	Root         root.Root
+	DeploymentId string
 }
 
-type GithubRepoCloneResponse struct {
+type DownloadRootResponse struct {
```

Comment:
I would just called it FetchRoot or GetRoot or ResolveRoot cuz this is doing more than downloading